lib/update-lockfile.js

```diff
@@ -32,7 +32,9 @@ module.exports = function updateLockfile (dependency, options) {
 
     if (options.yarn) {
       exec(`yarn add ${args}`)
-    } else {
+    }
+
+    if (options.npm) {
```

Comment:
Yep sorry, you're right about the options as I overlooked the `!!packageLock`. The other complication that arises from this is that the flags and prefixes are different for yarn and npm. This will need to be separated out if you want to update both.